client/galaxy/scripts/components/citations.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+    <div class="toolForm">
+        <div class="toolFormTitle">
+            Citations
+            <button v-if="viewRender" v-on:click="toggleViewRender" type="button" class="btn btn-xs" title="Show all in BibTeX format.">
+                <i class="fa fa-pencil-square-o"></i>
+                Show BibTeX
+            </button>
+            <button v-else type="button" v-on:click="toggleViewRender" class="btn btn-xs" title="Return to formatted citation list.">
+                <i class="fa fa-times"></i>
+                Hide BibTeX
+            </button>
+        </div>
+        <div class="toolFormBody citationPadding">
+            <div v-if="source === 'histories'" class="citation-padding">
+                <b>Warning: This is a experimental feature.</b> Most Galaxy
```

Comment:
I'll follow up in a subsequent PR and rework this block - I don't think this feature is so experimental anymore - most new tools have citations and we have done a good job back annotating as well.

Reply:
@jmchilton I'm happy to simplify that logic and remove the toggle right now.

Reply:
I like encouraging people to cite Galaxy though - that bit should stay ;).